fastlane/lib/fastlane/plugins/plugin_update_manager.rb

```diff
@@ -0,0 +1,65 @@
+module Fastlane
+  # Alert the user when updates for plugins are available
+  class PluginUpdateManager
+    def self.start_looking_for_updates
+      Thread.new do
+        self.references.each do |plugin_name, current_plugin|
+          begin
+            self.server_results[plugin_name] = Gem::Version.new(fetch_latest_version(plugin_name))
+          rescue
+          end
+        end
+      end
+    end
+
+    def self.show_update_status
+      # We set self.server_results to be nil
+      # this way the table is not printed twice
+      # (next to the summary table or when an exception happens)
+      return unless self.server_results.count > 0
+
+      rows = []
+      self.references.each do |plugin_name, current_plugin|
+        latest_version = self.server_results[plugin_name]
+        next if latest_version.nil?
+        current_version = Gem::Version.new(current_plugin[:version_number])
```

Comment:
We could also store versions here https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/plugins/plugin_manager.rb#L285 as `Gem::Version` instead of strings. 

Just a suggestion. No reason to fix it if its working great! 😄